requirements.txt

```diff
@@ -1,34 +1,27 @@
-#
-# These requirements were autogenerated by pipenv
-# To regenerate from the project's Pipfile, run:
-#
-#    pipenv lock --requirements
-#
-
 -i https://pypi.org/simple
-bracex==2.2.1; python_version >= '3.6'
```

Comment:
Does the loss of these `; python_version >= '3.6'` clauses matter? I suspect not.

Reply:
It didn't change the output of `make bs && make build`, so I guess not.